catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/TreUtility.java

```diff
@@ -73,4 +79,42 @@ public static Integer convertToInteger(Tre tre, String fieldName) {
         }
         return Optional.empty();
     }
+
+    public static Float convertToFloat(Tre tre, String fieldName) {
```

Comment:
I am willing to update but this will make this class inconsistent with what is already there (e.g. `public static Integer convertToInteger(Tre tre, String fieldName)`), and expand the scope into existing TREs that are not part of this ticket.